operator/config/crd/bases/customize.core.cnrm.cloud.google.com_namespacedcontrollerreconcilers.yaml

```diff
@@ -41,6 +41,20 @@ spec:
           spec:
             description: NamespacedControllerReconciler is the specification of NamespacedControllerReconciler.
             properties:
+              pprofConfig:
+                description: PprofConfigures the debug endpoint on the service.
```

Comment:
Also nit, should the comment should probably be "Configures the debug endpoint on the service" or "<fieldName> configures the debug endpoint on the service"